components/LiveControls.tsx

```diff
@@ -86,8 +86,11 @@ const main = () => {
 
     switch (data.type) {
       case "scrollToComponent": {
-        document
-          .getElementById(data.args.id)
+        (document
+            .getElementById(data.args.id) ?? data.args.alternateId
+          ? document
+            .getElementById(data.args.alternateId)
+          : undefined)
```

Comment:
I've changed that because the concept of `uniqueId` does not exists in the new engine. So I'm trying to support both behaviors providing an alternateId that is used as a section index

Reply:
Suggestion: Extract these conditions into consts.